app/views/admin/programs/_form.html.haml

```diff
@@ -8,5 +8,6 @@
       = f.input :schedule_public, label: "Show Schedule on the home and splash page"
       = f.input :schedule_fluid, label: "Allow submitters to change their event after it is scheduled"
       = f.input :rating, :hint => "Enter the number of different rating levels you want to have for voting on proposals. Enter 0 if you do not want to vote on proposals."
+      = f.input :languages, :hint => "Enter the languages allowed for events as values of #{link_to('ISO 639-1', 'http://www.loc.gov/standards/iso639-2/php/code_list.php', :target => "_blank")} language codes separated with commas. The first language would be the default language. Leave it blank if you do not want to specify languages.".html_safe
       %p.text-right
```

Comment:
@Ana06 There is a new coding style for ruby, which we try to follow in new code we write. Googling about it: https://github.com/bbatsov/ruby-style-guide#hash-literals